packages/Webkul/Core/src/Database/Migrations/2020_04_16_185147_add_table_addresses.php

```diff
@@ -267,11 +267,6 @@ private function migrateForeignKeys(): void
                         ->where('order_address_id', $row->additional['old_order_address_id'])
                         ->update(['order_address_id' => $row->id]);
                 });
-
-            $table->foreign(['order_address_id'])
-                ->references('id')
-                ->on('addresses')
-                ->onDelete('cascade');
```

Comment:
HI @devansh-webkul 

Create new migration file and remove from there, how this will work ?

Reply:
Creating new one.